src/domains/idpay-app/04_apim_api_mock_citizen_data.tf

```diff
@@ -0,0 +1,35 @@
+## IDPAY Mock citizen data ##
+module "idpay_citizen_data" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v6.15.2"
+
+  name                = "${var.env_short}-idpay-mock-citizen-data"
+  api_management_name = data.azurerm_api_management.apim_core.name
+  resource_group_name = data.azurerm_resource_group.apim_rg.name
+
+  description  = "IDPAY Mock citizen data"
+  display_name = "IDPAY Mock citizen data API"
+  path         = "idpay/mock/citizen"
+  protocols    = ["https", "http"]
+
+  service_url = "http://${var.ingress_load_balancer_hostname}/idpayadmissibility/idpay/mock/citizen"
+
+  content_format = "openapi"
+  content_value  = file("./api/idpay_mock_citizen_data/openapi.mock.citizen.yml")
+
+  xml_content = file("./api/base_policy.xml")
+
+  product_ids           = [data.azurerm_api_management_product.mock_api_product.product_id]
+  subscription_required = false
```

Comment:
have we tested the API without subscription key? The product module should enforce the api key requirement and therefore the `false` here may be misleading.